api/v1alpha1/clientprofile_types.go

```diff
@@ -25,6 +25,15 @@ import (
 type CephFsConfigSpec struct {
 	//+kubebuilder:validation:Optional
 	SubVolumeGroup string `json:"subVolumeGroup,omitempty"`
+
+	// Set mount options to use https://docs.ceph.com/en/latest/man/8/mount.ceph/#options
+	// Set to "ms_mode=secure" when connections.encrypted is enabled in Ceph
```

Comment:
@nb-ohad can you please remove this comment?

Reply:
done